.mk/vagrant.mk

```diff
@@ -36,22 +38,29 @@ vagrant-resume:
 vagrant-ssh:
 	@cd scripts/vagrant; vagrant ssh master
 
-.PHONY: vagrant-ssh-slave
-vagrant-ssh-worker:
-	@cd scripts/vagrant; vagrant ssh worker
+.PHONY: vagrant-ssh-worker%
+vagrant-ssh-worker%:
+	@cd scripts/vagrant; vagrant ssh worker$*
 
-.PHONY: vagrant-kublet-restart
-vagrant-restart-kublet:
-	@cd scripts/vagrant; vagrant ssh master -c "sudo service kubelet restart"; vagrant ssh worker -c "sudo service kubelet restart"
+.PHONY: vagrant-restart-kubelet
+vagrant-restart-kubelet:
+	@cd scripts/vagrant; vagrant ssh master -c "sudo service kubelet restart"; \
+	number=1 ; while [[ $$number -le ${WORKER_COUNT} ]] ; do \
```

Comment:
Coud this be `for`, mabe a bit more obvious?

Reply:
while or until loop seems more adapted than a "for in" loop: 
http://tldp.org/HOWTO/Bash-Prog-Intro-HOWTO-7.html

no?